app/views/metrics/review_turnaround.erb

```diff
@@ -0,0 +1,3 @@
+<div class='p-3'>
+  <%= line_chart Event.group(:created_at).count %>
```

Comment:
This is just as an example right? I'd add a comment here to clarify that this should be removed.